CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 Qiita 0.1.0-dev (changes since Qiita 0.1.0 go here)
 ---------------------------------------------------
 
+* Creating an empty RawData is no longer need in order to add a PrepTemplate. Now, the PrepTemplate is required in order to add a RawData to a study. This is the normal flow of an study, as the PrepTemplate information is available before the RawData information is available.
+* A user can upload a QIIME mapping file instead of a SampleTemplate. The system will create a SampleTemplate and a PrepTemplate from the information present in the QIIME mapping file.
```

Comment:
It would also be worth linking to the mapping file documentation on QIIME.

Also worth mentioning what are the columns they need to have in the QIIME mapping file for this to work as expected.